bin/pd-send

```diff
@@ -63,6 +63,10 @@ def build_queue_arg_parser(description):
         help="Add given KEY=VALUE pair to the event details"
         )
     parser.add_argument(
+        "-x", "--context", action="append", dest="contexts",
+        help="Add given TYPE:KEY=VALUE[,KEY=VALUE...] to the event contexts"
```

Comment:
@saj - we might change this to only key/value per option like in `--field` i.e. `--context TYPE:KEY=VALUE --context TYPE:KEY=VALUE` rather than `--context TYPE:KEY=VALUE,KEY=VALUE...`. This lets customers not worry about escaping commas or equal signs in a `VALUE`.